src/ontology/mondo-edit.obo

```diff
@@ -438119,8 +438119,8 @@ xref: Orphanet:827 {source="MONDO:equivalentTo"}
 xref: SCTID:47673003 {source="MONDO:equivalentTo"}
 xref: UMLS:C0271093 {source="MEDGEN:75734", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0003004 {source="DOID:0050817/inferred", source="MONDO:Redundant", source="NCIT:C85078"} ! macular degeneration
-is_a: MONDO:0005150 {source="DOID:0050817"} ! age-related macular degeneration
 is_a: MONDO:0016420 {source="Orphanet:827"} ! familial flecked retinopathy
+relationship: excluded_subClassOf MONDO:0005150 {source="PMID:10442900", source="PMID:11346402", source="https://orcid.org/0000-0002-0587-4693"} ! age-related macular degeneration
```

Comment:
@yshwetar the DOID should be added as a source on the excluded subclass as well, we want to keep the provenance that this came from DO